### PR TITLE
Added the Cross Origin API calls

### PR DESCRIPTION
**Pull request by pedro-acsousa**
Fixed the Cross Origin calls

### CODE REVIEW

**Comment by pedro-acsousa** [comment]
Added the cross origin calls to ensure jquery can call CORS

File: src/main/java/com/jlr/ttl/poc/loadmanager/controllers/VehicleController.java [hunk 1]
```diff
@@ -11,6 +11,7 @@
 import java.util.List;
 
 @RestController
+@CrossOrigin("http://localhost:4200")
```
Comment (BFARIAAL) [inline src/main/java/com/jlr/ttl/poc/loadmanager/controllers/VehicleController.java:15]: Missed import

**Review by BFARIAAL — CHANGES_REQUESTED** [review]
Resolve comments

**Comment by pedro-acsousa** [comment]
Bruno, I have added the status field to the DB file and the Vehicle entity as requested, via an enum class. 
Please review these changes. 
Additionally, the previous missing import has been resolved

**Review by BFARIAAL — CHANGES_REQUESTED** [review]
PR no longer in scope. Please remove the changes related to ticket #6 and raise them in their own PR

**Comment by pedro-acsousa** [comment]
Removed the extra commits

**Review by pedro-acsousa — COMMENTED** [review]
All has been resolved

**Pull request closed without merging**